.github/workflows/code-quality.yaml

```diff
@@ -15,14 +15,15 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Rerun all code generators we have
-        run: python3 ci/cached-builds/gen_gha_matrix_jobs.py
+        run: bash ci/generate_code.bash
```

Comment:
I don't like the `.bash` suffix :P , I can see we have it in one file already `./ci/cached-builds/gha_lvm_overlay.bash`... still, do we want to extend this policy?

Reply:
it's only tested with bash; when I get the mac, I will also test it on zsh

Reply:
What does the file suffix have to do with the which interpreter is used?